tools/sdk-generation-pipeline/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM mcr.microsoft.com/oss/mirror/docker.io/library/ubuntu:20.04
```

Comment:
Just FYI. This guy is timebombed. [Documentation](https://eng.ms/docs/more/containers-secure-supply-chain/definitions#timebombed-expiry)

So in 6 months or so you'll either need to send a request to get it renewed, or possibly transition to another one.

Reply:
thanks for the information. I have changed it to `mcr.microsoft.com/dotnet/runtime:3.1-focal`. Hope it meets the requirement.